superset/assets/cypress/integration/explore/visualizations/treemap.js

```diff
@@ -0,0 +1,59 @@
+describe('Treemap', () => {
+  const TREEMAP_FORM_DATA = {
+    datasource: '2__table',
+    viz_type: 'treemap',
+    slice_id: 50,
+    granularity_sqla: 'year',
+    time_grain_sqla: 'P1D',
+    time_range: '1960-01-01+:+now',
+    metrics: ['sum__SP_POP_TOTL'],
+    adhoc_filters: [],
+    groupby: ['country_code'],
+    row_limit: 50000,
+    color_scheme: 'bnbColors',
+    treemap_ratio: 1.618033988749895,
+    number_format: '.3s',
+  };
+
+  function verify(formData) {
+    cy.visitChartByParams(JSON.stringify(formData));
+    cy.verifySliceSuccess({ waitAlias: '@getJson', chartSelector: 'svg' });
+  }
+
+  beforeEach(() => {
+    cy.server();
+    cy.login();
+    cy.route('POST', '/superset/explore_json/**').as('getJson');
+  });
+
+  it('should work', () => {
+    verify(TREEMAP_FORM_DATA);
+    cy.get('.chart-container svg rect.child').should('have.length', 214);
```

Comment:
is this one robust to minor screen size changes? I thought I remembered treemap creating a variable # of rects based on available space, so wanted to make sure this wouldn't start failing if e.g., someone increased or decreased the size of the side panel.

Reply:
I checked the code and did not see any logic that reduce number of nodes. One component that does weird stuffs with the children is `icicle`, which only display top 5 children, but I don't see that with this treemap.